### DISCUSSION TITLE
Is there anyway to inject a laser pulse in 1 timestep?

### DISCUSSION THREAD

**Discussion by jiecaix** [BLAST-WarpX/warpx#4484] [Q&A] [answered]
I want to init a laser pulse, but just found the only way is laser antenna injection. Is there anyway to inject it in one timestep so that not a backward laser pulse generate?

**Comment by RemiLehe** [comment, marked as answer]
Thanks for the question.
There is no such option in WarpX currently. The only way to inject a laser is with the laser antenna.
